### ISSUE TITLE
Bug: REST API /events/restSearch ignores sort order

### ISSUE THREAD

**Issue by htioekb** [MISP/MISP#9359]
### Actual behavior

The following request tries to fetch events ordered by timestamp. The result is not sorted correctly.

```
POST /events/restSearch HTTP/1.1
{"limit":"10","page":"1","returnFormat":"json","timestamp":"1699012432","order": "Event.timestamp asc"}
```

The result is
```
{
    "response": [
        {
            "Event": {
                "id": "169563",
                "orgc_id": "5",
                "org_id": "1",
                "date": "2023-11-03",
                "threat_level_id": "3",
                "info": "Brand Abuse & Phishing - fereverhyt[.]com",
                "published": true,
                "uuid": "c17e4781-c026-40f9-a2c0-2a4e8312901d",
                "attribute_count": "1",
                "analysis": "2",
                "timestamp": "1699014675",
                "distribution": "3",
                "proposal_email_lock": false,
                "locked": true,
                "publish_timestamp": "1699016710",
                "sharing_group_id": "0",
                "disable_correlation": false,
                "extends_uuid": "",
                "protected": null,
                ...
            }
        },
        {
            "Event": {
                "id": "169564",
                "orgc_id": "573",
                "org_id": "1",
                "date": "2023-11-03",
                "threat_level_id": "1",
                "info": "[BeDisruptive] Ransomware BabLock",
                "published": true,
                "uuid": "6b1fb020-7c44-40ae-aab0-cdc92ae48dd9",
                "attribute_count": "8",
                "analysis": "2",
                "timestamp": "1699012432",
                "distribution": "3",
                "proposal_email_lock": false,
                "locked": true,
                "publish_timestamp": "1699016710",
                "sharing_group_id": "0",
                "disable_correlation": false,
                "extends_uuid": "",
                "protected": null,
                ...
            }
        }
    ]
}
```

### Expected behavior

Sending the REST call returns an order list of entries. The call

```
POST /events/restSearch HTTP/1.1
{"limit":"10","page":"1","returnFormat":"json","timestamp":"1699012432","order": "Event.timestamp asc"}
```

should result in

```
{
    "response": [
        {
            {
                "Event": {
                    "id": "169564",
                    "orgc_id": "573",
                    "org_id": "1",
                    "date": "2023-11-03",
                    "threat_level_id": "1",
                    "info": "[BeDisruptive] Ransomware BabLock",
                    "published": true,
                    "uuid": "6b1fb020-7c44-40ae-aab0-cdc92ae48dd9",
                    "attribute_count": "8",
                    "analysis": "2",
                    "timestamp": "1699012432",
                    "distribution": "3",
                    "proposal_email_lock": false,
                    "locked": true,
                    "publish_timestamp": "1699016710",
                    "sharing_group_id": "0",
                    "disable_correlation": false,
                    "extends_uuid": "",
                    "protected": null,
                    ...
                }
            },
            "Event": {
                "id": "169563",
                "orgc_id": "5",
                "org_id": "1",
                "date": "2023-11-03",
                "threat_level_id": "3",
                "info": "Brand Abuse & Phishing - fereverhyt[.]com",
                "published": true,
                "uuid": "c17e4781-c026-40f9-a2c0-2a4e8312901d",
                "attribute_count": "1",
                "analysis": "2",
                "timestamp": "1699014675",
                "distribution": "3",
                "proposal_email_lock": false,
                "locked": true,
                "publish_timestamp": "1699016710",
                "sharing_group_id": "0",
                "disable_correlation": false,
                "extends_uuid": "",
                "protected": null,
               ...
            }
        }
    ]
}
```

### Steps to reproduce

Send REST API call

```
POST /events/restSearch HTTP/1.1
{"limit":"10","page":"1","returnFormat":"json","timestamp":"1699012432","order": "Event.timestamp asc"}
```

I tried ordering by different attributes, but none worked. Same result when trying to order by attribute_count for example.
The sort order does not change when using asc or desc.

### Version

2.4.178

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

7.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by htioekb** [comment]
I think I found the bug.
In app/model/Event.php change line 2060 from
```
        if (!empty($options['order'])) {
            $options['order'] = $this->findOrder(
                $options['order'],
                'Event',
                ['id', 'info', 'analysis', 'threat_level_id', 'distribution', 'timestamp', 'publish_timestamp']
            );
        }
```

to

```
        if (!empty($options['order'])) {
            $params['order'] = $this->findOrder(
                $options['order'],
                'Event',
                ['id', 'info', 'analysis', 'threat_level_id', 'distribution', 'timestamp', 'publish_timestamp']
            );
        }
```

**Comment by righel** [comment]
Thanks for your report and digging into the solution @htioekb 
Pushed the fix into the `develop` branch: 76cc3ae844228adc1337479744095ed3832623e9
